control/control.leanos.xml

```diff
@@ -646,19 +649,15 @@ Please visit us at http://www.suse.com/.
                     <enable_next>yes</enable_next>
                 </module>
                 <module>
+                    <name>offline_migration</name>
+                    <enable_back>yes</enable_back>
+                </module>
+		<module>
                     <label>Product License</label>
                     <name>product_upgrade_license</name>
                     <enable_back>yes</enable_back>
                     <enable_next>yes</enable_next>
                 </module>
-                <module>
-                    <name>upgrade_urls</name>
```

Comment:
IIRC the `upgrade_urls` allows to reusing some repositories from the old system. Registration allows adding new repositories but it's quite difficult to enter the URLs again. With this client you enable the old repos with a single click.

Reply:
well, it is now quite broken as it shows old repos and also shows repos from scc. which looks strange. But it does not break any stuff, just looks strange. So I can readd it